msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -995,6 +995,41 @@ function toggleIncomingClientAudio(callback: (error: SdkError | null, result: bo
 
 ```
 
+### Example
+
+```Typescript
+callback = (error, result) => {
+        if (error) {
+            // Handle error code
+        }
+        else {
+            // Handle success code
+        }
+    }
+
+microsoftTeams.meeting.getIncomingClientAudioState(this.callback)
+
+```
+
+> [!NOTE]
+> RSC has been enabled after 1.11, this feature won't work before 1.12.
+
+### Manifest
```

Comment:
Not sure what we mean by Manifest in this context. Could you please check?
![image](https://user-images.githubusercontent.com/31851992/184101257-d6e75fd3-e36e-4b1d-afbd-7e27d30f1d41.png)

Reply:
Hello @Wajeed-msft - We wanted to add the section like we have at other places like below:
![image](https://user-images.githubusercontent.com/59052547/184120105-812cf409-be78-4107-8aba-6288093b2039.png)


@v-silakshmi - Can you please update it accordingly.

Reply:
![image](https://user-images.githubusercontent.com/59052547/184120435-9af41ee4-4872-4c23-a7b0-b8f37d69d9f5.png)